networks/ethereum/tokens/0xec53bF9167f50cDEB3Ae105f56099aaaB9061F83.json

```diff
@@ -0,0 +1,14 @@
+{
+  "CMCslug": "eigenlayer",
+  "decimals": 18,
+  "mldId": "ethereum-EIGEN-0xec53bF9167f50cDEB3Ae105f56099aaaB9061F83",
+  "orgName": "",
+  "orgUrl": "",
+  "tokenAddress": "0x0d8775f648430679a709e98d2b0cb6250d2887ef",
```

Comment:
this token address belongs to a different token.

Put the correct token address here as well